pytket/binders/mapping.cpp

```diff
@@ -106,6 +108,26 @@ PYBIND11_MODULE(mapping, m) {
       "Lexicographical Comparison approach outlined in arXiv:1902.08091.")
       .def(py::init<>(), "LexiLabellingMethod constructor.");
 
+  py::class_<
+      MultiGateReorderRoutingMethod,
+      std::shared_ptr<MultiGateReorderRoutingMethod>, RoutingMethod>(
+      m, "MultiGateReorderRoutingMethod",
+      "Defines a RoutingMethod object for commuting physically permitted "
+      "multi-qubit gates to the front of the subcircuit.")
+      .def(
+          py::init<unsigned, unsigned>(),
+          "MultiGateReorderRoutingMethod constructor.\n\n:param max_depth: "
+          "Maximum number of layers of gates checked for commutation, "
+          "\n:param max_size: Maximum number of gates checked for commutation.",
```

Comment:
Does this mean "simultaneous commutation"? I think so from a quick look at the code. Should clarify, otherwise it's not obvious what's meant.

Reply:
Yes, "simultaneous commutation" is more accurate.